config/types/update.go Added support for more fields in update section (#17)
flatcar-linux/container-linux-config-transpiler: doc/configuration.md

* **update**
    * **group** (string): the update group to follow. Most users will want one of: stable, beta, alpha.
    * **server** (string): the server to fetch updates from.

>     * **pcr_policy_server** (string): the server to revceive postede TPM PCR policy from.

Reviewer: ```suggestion
    * **pcr_policy_server** (string): the server to receive posted TPM PCR policy from.
```
but I wonder where you got this description from - I think the purpose is reverse ("the server to POST TPM PCR policy to")

    * **download_user** (string): the authentication user to fetch the update.

>     * **download_password** (string): the authentication user password/

Reviewer: ```suggestion
    * **download_password** (string): the authentication password to fetch the update
```

>     * **machine_alias** (string): human readable machine alias to be dsiplayed in the update server UI.

Reviewer: ```suggestion
    * **machine_alias** (string): human readable machine alias to be displayed in the update server UI.
```
